### PR TITLE
these are the steps to kick a locked machine

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: services/pul_kicker.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## File System Locks
+
+Occasionally servers will be locked down and we will need to kick them with a hard reboot. 
+
+The following steps will need a VPN connection
+
+* Using your preferred RDP Client ([Microsoft ships a good one](https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/remote-desktop-mac)) connect to `lib-staff998` -You will be prompted for you Active Directory netid and password.
+* On the resulting Desktop is a shortcut to the VSphere Application - Credentials are on lastpass-
+* Find the VM and restart or otherwise correct the mistake.
```
Comment (christinach) [inline services/pul_kicker.md:9]: The shortcut is VMware Vcenter.

Comment (christinach) [reply services/pul_kicker.md:9]: Clicking this shortcut takes you to `This site is not secure`
`Details -> Go on to the webpage` takes you a a different page where you need to select: `1.Vsphere webclient (flash)` or `2. vSphere client Html5 - partial functionality. `
Do we use on this step the credentials from Lastpass?

**Review by christinach — DISMISSED** [review]

File: services/pul_kicker.md (hunk 1 above)
Comment (christinach) [reply services/pul_kicker.md:9]: Which one should be used? option 1 or option2 ?

Comment (carolyncole) [reply services/pul_kicker.md:9]: HTML5 one should be used.  The flash one does not work.

Comment (carolyncole) [reply services/pul_kicker.md:9]: Also I had to search for the VM, browsing showed me no machines.  That is OK becuase I was on to reboot a single machine, but I almost left without doing anything thinking I did not have access

**Review by carolyncole — CHANGES_REQUESTED** [review]
I tried this out today to reboot lib-orange-prod6.
I added a couple comments as it was not entirely intuitive.

**Review by carolyncole — APPROVED** [review]
Works great for production and QA machines!

**Pull request merged**